modules/hcp-ibmz-update-reg-ca.adoc

```diff
@@ -0,0 +1,56 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-update-registry-ca-private-registry_{context}"]
+= Update Registry Certificate Authority in AgentServiceConfig with Mirror Registry
+
+When using a mirror registry for images, agents need to trust the registry's Certificate to securely pull images. For this, we must add the Certificate Authority of mirror registry to `AgentServiceConfig` by creating a `ConfigMap`.
+
```

Comment:
Add a `.Prerequisites` section to state that you must have installed the MCE Operator. When you refer to the MCE Operator, use the {mce} or {mce-short} attributes. For example:

```
.Prerequisites

* You must have installed {mce}.
```

Then, before the procedure begins, add a `.Procedure` heading per the modular documentation guidelines.